src/border_agent/border_agent.hpp

```diff
@@ -161,6 +163,10 @@ class BorderAgent : private NonCopyable
     std::map<std::string, std::vector<uint8_t>> mMeshCopTxtUpdate;
```

Comment:
Instead of introducing `mNonStandardTxtEntries`, I'd suggest to reuse `mMeshCopTxtUpdate`

Reply:
done!